docker-compose.yml

```diff
@@ -124,6 +124,17 @@ services:
     environment:
       - POSTGRES_URL=jdbc:postgresql://postgres/webknossos_testing
 
+  screenshot-tests:
+    build:
+      context: .
+      dockerfile: Dockerfile.puppeteer
+    environment:
+      - BRANCH
+    working_dir: /home/${USER_NAME:-pptruser}/webknossos
+    volumes:
+      - ".:/home/${USER_NAME:-pptruser}/webknossos"
+    user: ${USER_UID:-1000}:${USER_GID:-1000}
```

Comment:
I think it won't work to run this as somebody else than `pptruser`, because you need the special group rights. You can just remove the extra user handling, and make sure that the resulting images have `777` access rights (just `chmod` it in the yarn target, as there it's still the same user).